test/code/plugins/oms_common_test.rb

```diff
@@ -172,11 +172,17 @@ def test_get_os_version_missing_field()
 
     def test_get_hostname
       $log = MockLog.new
+      file_hostname = '/etc/hostname'
```

Comment:
added tests to verify scenarios when host file exist and when doesn't.
please note that the container file host path is changed to /etc/containerhostname